src/App.tsx

```diff
@@ -1,29 +1,36 @@
-import { Component } from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 
-interface State {
-  movies: Movie[];
-}
+export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [inList, setInList] = useState(false);
```

Comment:
btw, do you need it in this component?

```suggestion
  const [isInList, setIsInList] = useState(false);
```